src/api/tokenList/Faucet_info.md

```diff
@@ -0,0 +1,51 @@
+# Ether, Tether, TrueUSD, USD Coin, Paxos, Gemini, Dai
```

Comment:
Note that I tried to capture this information inside the json as well by adding *website*, *description* and *faucet* fields.